update import user instructions (#2751)
cloud-gov/product: .github/ISSUE_TEMPLATE/pages-3-zendesk-sync.md

### User Import

- Download and Import the Users CSV into Google Sheets for editing purposes

> - For "Sperator type" keep it unchanged from "Detect automatically" 

Reviewer: ```suggestion
- For "Seperator type" keep it unchanged from "Detect automatically" 
```

- Delete the columns "Created" and "Last Signed In"
- Delete all of the content in "ID"

> - The remaning colums should be "ID", "Email", "Organizations" and "Details"

Reviewer: ```suggestion
- The remaining columns should be "ID", "Email", "Organizations" and "Details"
```

*The "Name" and "Email" fields are required for importing new users, without them the entire job will fail*

> - Copy and paste the entire "Email" column into the blank column (A)

Reviewer: Does this replace the "ID" column?

Author: Yep just replacing everything in that column.

Reviewer: Maybe 

```suggestion
- Copy and paste the entire "Email" column into the blank column (A) that used to be the "ID" column
```

for clarity?

- Rename the header to "Name" from "Email"
- Utilize the "Find and replace" function by hitting command + shift + H
- In search choose "Specific range" from the dropdown and specify A:A as the range parameter
- In "Find" specify all email addresses that need to be replaced and leave "Replaced with" null

> (ie. Find:@gsa.gov Replace with: )

Reviewer: ```suggestion
(i.e. Find: "@gsa.gov" Replace with: "")
```

- This will remove all the email address from the names under the "Name" header making it easier to format for importing purposes

> - Replace all "." with an " ," there is a space then comma

Reviewer: ```suggestion
- Replace all "." with an " ," — that is, a space then a comma
```

> (ie. pages.user -> pages, user)

Reviewer: ```suggestion
(i.e. from "pages.user" to "pages, user")
```

- Save the sheet as a CSV
- Navigate to the Zendesk admin center 
- In the lefthand panel locate "Bulk actions" and choose "Import organizations"

> - An import job report will be sent to your .gov inbox where you can view if the job was successful/unsuccesful

Reviewer: ```suggestion
- An import job report will be sent to your .gov inbox where you can view if the job was successful/unsuccessful
```

> **Note** In the csv you will notice a number of users emails towards the bottom which do not have the organization or details field associated with them and will consequently result as failures in the report results as such

Reviewer: ```suggestion
**Note** In the CSV you will notice a number of users' emails towards the bottom which do not have the organization or details field associated with them and will consequently result as failures in the report results. For example:
```
